content/plugins/codewithdennis-filament-select-tree.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Select Tree
+slug: codewithdennis-select-tree
+author_slug: codewithdennis
+categories: [form-field]
+description: The multi-level select field lets you pick one or multiple options from a list that's neatly organized into different levels.
+discord_url: #
+docs_url: https://github.com/CodeWithDennis/filament-select-tree/blob/3.x/README.md
+github_repository: codewithdennis/filament-select-tree
```

Comment:
Please use the raw github url for your docs_url.

https://raw.githubusercontent.com/CodeWithDennis/filament-select-tree/3.x/README.md

Reply:
Done @awcodes ✅